velox/dwio/parquet/reader/ParquetData.cpp

```diff
@@ -143,4 +210,59 @@ std::pair<int64_t, int64_t> ParquetData::getRowGroupRegion(
   return {fileOffset, length};
 }
 
+std::shared_ptr<BloomFilter> ParquetData::getBloomFilter(
+    dwio::common::BufferedInput& bufferedInput,
+    const uint32_t rowGroupId,
+    const uint32_t column) {
```

Comment:
No need to pass column parameter. The column information is already in this object.